src/manageiq_client/api.py

```diff
@@ -622,6 +622,15 @@ def _process_result(self, result):
             if "id" in result:
                 d["href"] = "{}/{}".format(collection._href, result["id"])
             return Entity(collection, d)
+        elif "task_href" in result or "task_id" in result:
+            collection = getattr(self.api.collections, "tasks")
+            # reuse task_href and task_id, no other data is relevant
+            d = {}
+            if result.get("task_href"):
```

Comment:
nit: `if "task_href" in result:` (more idiomatic and consistent with surrounding code)